common/lib/xmodule/xmodule/course_module.py

```diff
@@ -380,6 +380,20 @@ class CourseFields(object):
         ),
         scope=Scope.settings
     )
+    enable_ccx = Boolean(
+        # Translators: Custom Courses for edX (CCX) is an edX feature for re-using course content. CCXs allow course
+        # staff to run some or all of an existing course for a group of students on a new schedule.
+        display_name=_("Enable CCX"),
+
+        # Translators: CCX Coach is a role created by a course Instructor. All CCX features are configurable in edX LMS
+        # by CCX Coach.
```

Comment:
This is good but translators may not translate strings sequentially or different translators may translate different strings. So every string that mentions CCX should explain what CCX is.